tutorials/underwater_vehicles.md

```diff
@@ -14,7 +14,7 @@ The behaviour of a moving body through water is different from the behaviour of
 a ground based vehicle. In particular bodies moving underwater experience much
```

Comment:
I noticed that this tutorial isn't listed on `tutorials.md.in`, can we add it there?

Also please add `\page underwater_vehicles Underwater vehicles` to the top so the tutorial is correctly generated by doxygen. Thanks!